framework/src/media/audio/audio_manager.c

```diff
@@ -1459,6 +1483,10 @@ audio_manager_result_t find_stream_in_device_with_process_type(device_process_ty
 					continue;
 				}
 
+				caps_desc.caps.ac_len = sizeof(struct audio_caps_s);
```

Comment:
This is always same, we don't have to move this, because we should find card & device number with given values.

Reply:
This needs to be moved because ac_type and ac_subtype are changed in wm1840 driver.
Or this function operates incorrectly.